logstash-core/spec/logstash/filter_delegator_spec.rb

```diff
@@ -92,6 +93,7 @@ def filter(event)
       end
 
       it "increments the in/out of the metric" do
+        allow(metric).to receive(:increment).with(:time_ms, anything)
```

Comment:
I think maybe I miswrote the fix here. THis `allow` belongs in a `before` statement. It's not really part of the test, just saying that such a call is OK.

Maybe I could expand it to allow *any* `:increment` calls and we just assert that `:in` and `:out` are at least invoked, because that's what the test really expects, imo.